compaction/compaction.cc

```diff
@@ -1717,14 +1717,17 @@ static future<compaction_info> scrub_sstables_validate_mode(sstables::compaction
         }
     });
 
-    clogger.info("Scrubbing in validate mode {}", sstables_list_msg);
+    clogger.info("Scrubbing in validate mode: {}", sstables_list_msg);
 
     auto permit = cf.compaction_concurrency_semaphore().make_tracking_only_permit(schema.get(), "scrub:validate", db::no_timeout);
     auto reader = sstables->make_crawling_reader(schema, permit, descriptor.io_priority, nullptr);
 
     const auto valid = co_await scrub_validate_mode_validate_reader(std::move(reader), *info);
+    if (!valid) {
+        info->invalid_sstables++;
```

Comment:
```
info->invalid_sstables += sstables.size();
```
Should scrub proper also bump this?

Reply:
> `info->invalid_sstables += sstables.size();`

Are the sstables merged so we don't know which of them was invalid?
(that said, it seems like we're passing a single sstable at a time anyhow.)

> Should scrub proper also bump this?

Yes

Reply:
> > `info->invalid_sstables += sstables.size();`
> 
> Are the sstables merged so we don't know which of them was invalid?

Yes there is no way to tell

> (that said, it seems like we're passing a single sstable at a time anyhow.)

Correct, but we cannot assume this as we get a `vector<sstable>` here.